.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 defaults: &defaults
   working_directory: /go/src/github.com/gruntwork-io/cloud-nuke
   docker:
-    - image: 087285199408.dkr.ecr.us-east-1.amazonaws.com/circle-ci-test-image-base:go1.13
+    - image: 087285199408.dkr.ecr.us-east-1.amazonaws.com/circle-ci-test-image-base:go1.15
```

Comment:
I don't believe we've published such an image yet.

Reply:
@brikis98 just keeping up with tooling. I didn't realize that an intermediate image is a build dependency.